pkg/command/completion_helper.go

```diff
@@ -9,9 +9,10 @@ import (
 
 const (
 	// Completion strings for the values of the --target flag
-	compK8sTarget = "k8s\tFor interactions with a Kubernetes cluster"
-	compTAETarget = "tae\tFor interactions with a Application Engine endpoint"
-	compTMCTarget = "tmc\tFor interactions with a Mission-Control endpoint"
+	compGlobalTarget = "global\tApplicable globally"
+	compK8sTarget    = "k8s\tFor interactions with a Kubernetes cluster"
+	compTAETarget    = "tae\tFor interactions with a Application Engine endpoint"
+	compTMCTarget    = "tmc\tFor interactions with a Mission-Control endpoint"
```

Comment:
This might create possible conflicts with https://github.com/vmware-tanzu/tanzu-cli/pull/524/files#diff-abbc3797392e0d0f203b1be8753c9886d91ea6ce318349575f79a1840ca506fbL12-R14.

Adding the comment here to link it when one PR gets in another will need to be updated.